scripts/var_props.py

```diff
@@ -874,6 +874,10 @@ def __init__(self, var1_stdname, var1_type, var1_kind, var1_units,
         self.__v2_context = v2_context
         self.__v1_kind = var1_kind
         self.__v2_kind = var2_kind
+        self.v1_units = var1_units
```

Comment:
I don't see this newly added properties being used?

Reply:
They are used in suite_objects.py, https://github.com/NCAR/ccpp-framework/pull/604/files#diff-dc7d94eadf868af215e91f5fe0fbbd4257a53528224e30aeb5ae27aab33e1500R1671